app/frontend/helpers/i18n_helper.js

```diff
@@ -83,17 +83,13 @@ class I18nHelper {
   }
 
   async getSpecificTranslation(lang, key) {
-    var origLang = i18n.language;
-
-    await i18n.changeLanguage(lang);
-    var specificTranslation = i18n.t(key);
-    await i18n.changeLanguage(origLang);
+    var specificTranslation = i18n.t(key, {lng: lang}); // https://www.i18next.com/translation-function/essentials
 
     return specificTranslation;
   }
 
   async getChapterTranslation(lang) {
-    var language = lang;
+    var language = lang??this.getLanguage();
```

Comment:
I am not sure whether the `??` operator works with browserify (needed for Android app). That's a newer JavaScript feature, isn't it? I recently played with the new optional chaining (`?.`) operator and that turned out to not work with browserify ...